extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.js

```diff
@@ -130,4 +135,36 @@ export class AmpStoryInteractiveImgQuiz extends AmpStoryInteractive {
 
     return convertedOption;
   }
+
+  /**
+   * @override
+   */
+  displayOptionsData(optionsData) {
+    if (!optionsData) {
+      return;
+    }
+
+    const percentages = this.preprocessPercentages_(optionsData);
+
+    this.getOptionElements().forEach((el, index) => {
+      // Update the aria-label so they read "selected" and "correct" or "incorrect"
+      const ariaDescription = objstr({
+        selected: optionsData[index].selected,
+        correct: el.hasAttribute('correct'),
+        incorrect: !el.hasAttribute('correct'),
+      });
+      el.setAttribute(
+        'aria-label',
+        ariaDescription + ' ' + this.options_[index]['imagealt']
+      );
+      el.querySelector(
+        '.i-amphtml-story-interactive-img-quiz-answer-choice'
+      ).setAttribute('aria-hidden', true);
```

Comment:
We will need to check how readers use text elements inside buttons, since we're going to add the aria-label attribute to the button, we might not need to worry about the aria-label/hidden inside it

Reply:
Removed in new commit